components/prices/SpecialPriceCard.tsx

```diff
@@ -65,12 +61,12 @@ export default function SpecialPriceCard({
           popular ? '' : 'hidden'
         } flex flex-col items-center justify-center py-2 text-sm font-medium text-white`}
       >
-        {popular && billingFrequency === 12 ? contentText.cta.discount + ' ' + storage : contentText.mostPopular}
+        {popular && billingFrequency === 'year' ? contentText.cta.discount + ' ' + storage : contentText.mostPopular}
```

Comment:
This check is duplicated, extract it as one variable/function whatever, and do use enums if possible, instead of strings. If you change the string, you will have to change it on the entire app.